modules/ROOT/pages/runtime-installation-task.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Before downloading and installing Mule runtime, verify that you have Java SE JDK 1.8 installed, for example:
+Before downloading and installing Mule, verify that you have Java SE JDK 1.8 installed, for example:
```

Comment:
Why just Mule and not Mule Runtime?

Reply:
Hi Pablo,
We are supposed to use the correct product names. So it is either just Mule, or Mule runtime engine. See https://docs.mulesoft.com/general/getting-started/glossary#m